### PR TITLE
Add API call to configure beam divergence

### PR DESCRIPTION
**Pull request by prybicki**

### CODE REVIEW

File: include/rgl/api/core.h [hunk 1]
```diff
@@ -713,6 +713,15 @@ RGL_API rgl_status_t rgl_node_raytrace_configure_non_hits(rgl_node_t node, float
...
 RGL_API rgl_status_t rgl_node_raytrace_configure_mask(rgl_node_t node, const int8_t* rays_mask, int32_t rays_count);
 
+/**
+ * Modifies RaytraceNode to set beam divergence.
+ * Beam divergence is used to calculate the beam width at the distance of hit point.
+ * Setting beam divergence > 0.0f is required to use query for multi-return results.
+ * @param node RaytraceNode to modify.
+ * @param beamDivergence Beam divergence in radians.
+ */
+RGL_API rgl_status_t rgl_node_raytrace_configure_beam_divergence(rgl_node_t node, float beamDivergence);
```
Comment (PawelLiberadzki) [inline include/rgl/api/core.h:723]:
```suggestion
RGL_API rgl_status_t rgl_node_raytrace_configure_beam_divergence(rgl_node_t node, float beam_divergence);
```
I assume snake case should be used here. And `rgl_node_raytrace_configure_non_hits` just has the same mistake.

File: include/rgl/api/core.h [hunk 2]
```diff
@@ -713,6 +713,15 @@ RGL_API rgl_status_t rgl_node_raytrace_configure_non_hits(rgl_node_t node, float
  */
 RGL_API rgl_status_t rgl_node_raytrace_configure_mask(rgl_node_t node, const int8_t* rays_mask, int32_t rays_count);
 
+/**
+ * Modifies RaytraceNode to set beam divergence.
+ * Beam divergence is used to calculate the beam width at the distance of hit point.
+ * Setting beam divergence > 0.0f is required to use query for multi-return results.
```
Comment (PawelLiberadzki) [inline include/rgl/api/core.h:719]: Can I maybe set beam divergence to 0.0f through this call, in order to disable it? This partially refers to line 949 in apiCore.cpp, where you check for `divergence >= 0.0f`.

**Review by PawelLiberadzki — APPROVED** [review]

File: include/rgl/api/core.h (hunk 2 above)
Comment (prybicki) [reply include/rgl/api/core.h:719]: Yes, that was my intention. I'll mention this in the documentation, thanks.

File: include/rgl/api/core.h (hunk 1 above)
Comment (prybicki) [reply include/rgl/api/core.h:723]: Yes, you are correct, thanks for the catch.

**Pull request merged**